Packs/Gmail/Integrations/Gmail/Gmail.yml

```diff
@@ -61,6 +73,16 @@ script:
     description: Deletes a Gmail user.
     execution: false
     name: gmail-delete-user
+    outputs:
+    - contextPath: GmailLabel.UserID
```

Comment:
Maybe instead of ```GmailLabel.UserID``` name it ```Gmail.Labels.UserID```? I see this is the convention in this integration.

Reply:
I looked through the existing code. The Gmail.Labels shows labels that are a property of a specific e-mail. These are just a list so I didn;t want to confuse the user with something that looked like it may belong to a specific e-mail.